pkg/plugin/predicates/predicate_manager.go

```diff
@@ -23,9 +23,9 @@ import (
 	"fmt"
 
 	"go.uber.org/zap"
+	admissionV1 "k8s.io/api/admission/v1"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/kube-scheduler/config/v1beta1"
```

Comment:
I think this should be "k8s.io/kube-scheduler/config"...

Reply:
This is not related to the admission controller either. This is linked to the plugins. probably should not be part of this change either.